python/lsst/ctrl/execute/configurator.py

```diff
@@ -220,8 +222,8 @@ def createConfiguration(self, input):
         @return the newly created Orca configuration file
         """
         resolvedInputName = envString.resolve(input)
-        if self.opts.verbose == True:
-            print "creating configuration using ",resolvedInputName
+        if self.opts.verbose is True:
```

Comment:
is the `is True` needed at all here? Can we just use `verbose` in boolean context?